tests/integration/test_verify.py

```diff
@@ -16,6 +17,24 @@ def test_verify_valid_sources(source_dir):
     assert exit_code == 0
 
 
+@pytest.mark.parametrize('source_dir,output', [
+    ("tests/test_files/yaml_source_dir/valid_yamls_with_multiple_crds",
+     "tests/bundle.yaml"),
+])
+def test_verify_with_output_file(source_dir, output):
+    process = subprocess.Popen(f'operator-courier verify {source_dir} --output={output}',
```

Comment:
Why do you need popen? Can you just use `api` function in unittest instead? (AFAIK there are no cli tests in operator-courier which should handle arg parsing and passing)

Reply:
This isn't a unit test. This is an integration test that ensures that, on the system, if you use the cli tool with this flag that a file actually gets created. We could also create a unit test that tests this code is executed, but that's not the purpose here.